__tests__/index.test.ts

```diff
@@ -2281,106 +2323,106 @@ describe('Auth0', () => {
       expect(fn).toThrow();
     });
   });
-});
 
-describe('default creation function', () => {
-  it('does nothing if there is nothing in storage', async () => {
-    jest.spyOn(Auth0Client.prototype, 'getTokenSilently');
-    const getSpy = jest
-      .spyOn(require('../src/storage').CookieStorageWithLegacySameSite, 'get')
-      .mockReturnValueOnce(false);
+  describe('default creation function', () => {
+    it('does nothing if there is nothing in storage', async () => {
+      jest.spyOn(Auth0Client.prototype, 'getTokenSilently');
+      const getSpy = jest
+        .spyOn(require('../src/storage').CookieStorageWithLegacySameSite, 'get')
+        .mockReturnValueOnce(false);
```

Comment:
The tests are starting to look better, can you please also remove any unnecessary whitespace and formatting changes?

Reply:
This should be fixed in 34964af

Reply:
Yes looks great!